Classes/Settings/SettingsAccountsViewController.swift

```diff
@@ -62,10 +70,13 @@ final class SettingsAccountsViewController: UITableViewController, GitHubSession
         present(alert, animated: trueUnlessReduceMotionEnabled)
     }
 
-    private func handleError() {
+    private func handleError(_ errorString: String = "") {
+        
+        let alertString = errorString.count > 0 ? errorString : "There was an error adding another account. Please try again."
+        
         let alert = UIAlertController.configured(
             title: NSLocalizedString("Error", comment: ""),
-            message: NSLocalizedString("There was an error adding another account. Please try again.", comment: ""),
+            message: NSLocalizedString(alertString, comment: ""),
```

Comment:
Here we don't really need `NSLocalizedString()` because string value is determined on run time